features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/app/components/Apis/Details/Overview.js

```diff
@@ -31,10 +31,10 @@ import Grid from 'material-ui/Grid';
 import Typography from 'material-ui/Typography';
 import Button from 'material-ui/Button';
 import TextField from 'material-ui/TextField';
-import Card, { CardActions, CardContent, CardMedia } from 'material-ui/Card';
-import Table, { TableBody, TableCell, TableRow } from 'material-ui/Table';
-import { Delete, Edit, CreateNewFolder, Description  }from 'material-ui-icons';
-import Tabs, { Tab } from 'material-ui/Tabs';
+import Card, {CardActions, CardContent, CardMedia} from 'material-ui/Card';
+import Table, {TableBody, TableCell, TableRow} from 'material-ui/Table';
+import {Delete, Edit, CreateNewFolder, Description}from 'material-ui-icons';
```

Comment:
This will make the bundle js heavy. Do we have configured for Tree-shaking ?

Reply:
Yes we [do](https://github.com/wso2/carbon-apimgt/blob/master/features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/package.json#L8),  in npm production build script we have used `-p` webpack [CLI argument](https://webpack.js.org/guides/production/#cli-alternatives) to do the [minification + code optimization](https://webpack.js.org/guides/tree-shaking/#minify-the-output) including DCE and various other optimizations.